src/cashweb/relay/index.ts

```diff
@@ -520,6 +521,9 @@ export class RelayClient extends ReadOnlyRelayClient {
         if (item.type === 'text') {
```

Comment:
This block needs to be factored out into another function that can be called by `constructForwardEntry`